server/src/routes/router.js

```diff
@@ -0,0 +1,19 @@
+import user from '../Users/UserController';
```

Comment:
We should be more specific about our imported variables when we have both controller and model. Elsewhere, the user model is imported as 'User'. Here, maybe we should import as 'userController'?